data/bash-completion/fwupdmgr

```diff
@@ -7,6 +7,10 @@ _fwupdmgr_cmd_list=(
 	'downgrade'
 	'download'
 	'enable-remote'
+	'emulate-add'
+	'emulate-remove'
+	'emulate-load'
+	'emulate-save'
```

Comment:
need fwupdtool too, no?

Reply:
Got it, thanks.

Reply:
Wait, but you have emulate-load and emulate-save commands in fwupdtool..

Reply:
This was more to test the loading and saving of zipfiles; I think the only thing it can capture is setup.json *if* you've already done fwupd**mgr** emulate-add. Maybe remove it because of the limitations and because it's confusing?

Reply:
I guess I would agree it would make more sense to tear out everything from fwupdtool then.

Reply:
Ack, done.